WcaOnRails/app/views/competitions/edit.html.erb

```diff
@@ -122,6 +122,37 @@
         });
       </script>
 
+
+      <% if @competition_admin_view %>
```

Comment:
I'm thinking "automated test". Let's screenshare sometime and I'll explain what I mean.

Can we show 1 of these 3 messages based on which state we're in:

- There are no nearby competitions!
- You have not assigned a date for this competition yet.
- You have not assigned a location for this competition yet.